### PR TITLE
ENDOC-188 Improve react tutorial

### PR DESCRIPTION
**Pull request by nshaw**

### CODE REVIEW

**Comment by nshaw** [comment]
@raxkaynan , see if that's clearer when you get a chance.

File: vuepress/docs/next/tutorials/micro-frontends/react.md [hunk 1]
```diff
@@ -216,12 +216,12 @@ Example:
 PUBLIC_URL=http://quickstart-entando.192.168.64.34.nip.io/entando-de-app/cmsresources/my-widget
 ```
 
-3. Replace `quickstart-entando.192.168.64.34.nip.io/app-builder` with the URL for your Entando App Builder instance. → [How to find your Entando App Builder URL.](../../docs/getting-started/#deploy-entando)
+3. Replace `quickstart-entando.192.168.64.34.nip.io/entando-de-app` with the URL for your Entando application.
```
Comment (joewhite101) [inline vuepress/docs/next/tutorials/micro-frontends/react.md:219]: This (and the line above it) are incorrect. We should be using a relative path there or passing a path as an argument to the MFE unless this is for local dev.

**Review by joewhite101 — CHANGES_REQUESTED** [review]
Not directly related to this PR but we should fix if we're changing these lines. I don't think the setup is what we should be encouraging here.

File: vuepress/docs/next/tutorials/micro-frontends/react.md (hunk 1 above)
Comment (nshaw) [reply vuepress/docs/next/tutorials/micro-frontends/react.md:219]: Good call. I never liked that this broke running the app locally too. Will update to .env.production with a relative url.

Comment (nshaw) [reply vuepress/docs/next/tutorials/micro-frontends/react.md]: Updated

File: vuepress/docs/v6.3/tutorials/micro-frontends/react.md [hunk 2]
```diff
@@ -196,36 +193,26 @@ If you're getting started with a new install of Entando, let's add our widget to
...
-
 ::: warning Notes
-- `quickstart-entando.192.168.64.34.nip.io` represents your `Entando App Builder` instance.
-- `/entando-de-app/cmsresources/` is your Resource URL
-- `my-widget` is the public folder we'll create to host our files
-:::
-
-::: tip
-[When you run `npm run build`, `Create React App` will substitute `%PUBLIC_URL%` with a correct absolute path so your project works even if you use client-side routing or host it at a non-root URL.](https://create-react-app.dev/docs/using-the-public-folder/)
+- `/entando-de-app/cmsresources/` is the Resource URL for your Entando application
```
Comment (joewhite101) [inline vuepress/docs/v6.3/tutorials/micro-frontends/react.md:214]: capitalized Resource

Comment (joewhite101) [reply vuepress/docs/v6.3/tutorials/micro-frontends/react.md:214]: disregard

**Review by joewhite101 — APPROVED** [review]
GTM

**Pull request merged**